website/notifications/emails.py

```diff
@@ -78,7 +80,17 @@ def email_digest(subscribed_user_ids, uid, event, **context):
 }
 
 
-def notify(uid, event, **context):
+def notify(uid, event, user, node, timestamp, **context):
+    """
+    :param uid: node's id
+    :param event: type of notification
+    :param user: user triggering notification
+    :param node: the node
+    :param timestamp: time
+    :param context: optional variables specific to templates
+        target_user: used with comment_replies
```

Comment:
Might make more sense to document these variables in the templates that use them.